content/sensu-core/1.5/quick-start/five-minute-install.md

```diff
@@ -148,52 +174,29 @@ Sensu Core users:
    }
  }' |sudo tee /etc/sensu/uchiwa.json{{< /highlight >}}
 
-Sensu Enterprise users:
-
-{{< highlight shell >}}
-echo '{
-  "sensu": [
-    {
-      "name": "sensu",
-      "host": "127.0.0.1",
-      "port": 4567
-    }
-  ],
-  "dashboard": {
-    "host": "0.0.0.0",
-    "port": 3000
-  }
-}' |sudo tee /etc/sensu/dashboard.json{{< /highlight >}}
-
-**9. Make sure that the `sensu` user owns all of the Sensu configuration files:**
+**10. Make sure that the `sensu` user owns all of the Sensu configuration files:**
 
 {{< highlight shell >}}
 sudo chown -R sensu:sensu /etc/sensu{{< /highlight >}}
 
-**10. Start the Sensu services**
-
-Sensu Core users:
+**11. Start the Sensu services:**
 
 {{< highlight shell >}}
 sudo systemctl enable sensu-{server,api,client}
 sudo systemctl start sensu-{server,api,client}
 sudo systemctl enable uchiwa
 sudo systemctl start uchiwa{{< /highlight >}}
 
-Sensu Enterprise users:
-
-{{< highlight shell >}}
-sudo systemctl enable sensu-{enterprise,enterprise-dashboard,client}
-sudo systemctl start sensu-{enterprise,enterprise-dashboard,client}{{< /highlight >}}
+Nice work! You have successfully installed and configured Sensu.
 
-**11. Verify that your installation is ready to use by querying the Sensu API
-    using the `curl` utility (and piping the result to the [`jq` utility][10]):**
+You can verify that your installation is ready to use by querying the Sensu API
+using the `curl` utility (and piping the result to the [`jq` utility][10]):
 
 {{< highlight shell >}}
 sudo yum install jq curl -y
 curl -s http://127.0.0.1:4567/clients | jq .{{< /highlight >}}
 
-If the Sensu API returns a JSON array of Sensu clients similar to this:
+The Sensu API should return a JSON array of Sensu clients similar to this:
 
 {{< highlight shell >}}
 $ curl -s http://127.0.0.1:4567/clients | jq .
```

Comment:
Unrelated to these changes, but this command is repeated from line 194. Maybe remove one? Not sure which makes the most sense.